src/platform/mbed/BUILD.gn

```diff
@@ -0,0 +1,24 @@
+import("//build_overrides/chip.gni")
+import("${chip_root}/src/platform/device.gni")
+
+assert(chip_device_platform == "mbed")
+
+source_set("mbed_bsd_socket") {
+    public_configs = [ "${chip_root}/src:includes" ]
+
+    sources = [
+        "${chip_root}/src/platform/mbed/net/src/BSDSocket.cpp",
```

Comment:
nit: just "src/BSDSocket.cpp"

Reply:
Thanks, I made the change and rebased against `master` as CI was failing.